core/controllers/admin_test.py

```diff
@@ -112,57 +110,3 @@ def test_change_about_page_config_property(self):
 
         response = self.testapp.get('/about')
         self.assertIn(new_config_value, response.body)
-
-    def test_change_rights(self):
```

Comment:
It's no longer relevant, because we no longer use Jinja to populate these values. I considered doing an e2e test for it, but decided against this because the functionality it tests for is relatively minor and our current strategy for the e2e tesfs is to test high-level general behaviour rather than small details.